### PR TITLE
Edits to Git entries (11 total)

### LINKED ISSUE #538: [Edit] Subject : Git
### Type of Edit (select all that apply)

Fix grammar, spelling, Clarify unclear content and/or terms, Add new content (definitions, codeblocks, etc.)

### Description (optional)

There are several incorrect, incomplete or unclear areas in the Git content.
This issue is to edit/update those.  No new Git concepts to be added as part of this issue.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by aedwardg**
### Description

Closes #538 

Update incorrect, incomplete or unclear areas in the Git content.
Change some first and second person language to the third person as requested here:
https://github.com/Codecademy/docs/issues/538#issuecomment-1039460169

### Type of Change

- Editing existing entries (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by Dusch4593** [comment]
Hey @aedwardg! Thank you for this edit and I'd be happy to take a look in a little bit. 

Just a heads up, though: for future reference, could we not include the relevant issue number within the name of the PR? It makes things a little difficult to track on our end with those numbers so close together. Linking under "Labels" on the right and writing "Closes #[issue-number]" is sufficient.

File: content/git/concepts/add/add.md
Comment (Dusch4593) [inline content/git/concepts/add/add.md:57]: I'm beginning to wonder if using an unordered list makes more sense? This is, after all, just a list of commonly used flags for `git add`, not a sequence of steps or anything like that.

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [inline content/git/concepts/branch/branch.md:39]:
```suggestion
Sometimes, Git will refuse to delete a branch because it has changes that have not yet been merged into an existing upstream or remote version of that branch. In such cases, the branch can be forcefully deleted with either the `-d --force` or capital `-D` flags, like so: 
```

Our colleagues in Product prefer to be as minimal with emphasis as possible, which includes leaving out italics in almost all cases.

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [inline content/git/concepts/branch/branch.md:41]:
```suggestion
```pseudo
```

Comment (Dusch4593) [reply content/git/concepts/branch/branch.md:41]: Since we're using placeholders for real values like `<branch-name>`, let's use ```pseudo for those cases.

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [inline content/git/concepts/branch/branch.md:46]:
```suggestion
# Option 1:
$ git branch -D my-branch

# Option 2:
$ git branch -d --force my-branch
```

After the syntax of a command is introduced, instead of using `<placeholder-syntax>` let's switch to using fictional names like `my-branch`.

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [inline content/git/concepts/branch/branch.md:53]: A couple a thoughts to note about lines #53-57:

* Let's change the sentence in line #53 to exclude "basic" from it.
* On lines #55-57, let's change `<branch-name>` to `branch-name` to make the pseudocode syntactically compliant w/ Git (running with `<`,`>` yields an error).
* * Since this section talks about moving between branches, should we include information about [the `git switch` command](ff3e4e1bce8bcc8068623639aa5f0272e6e6c49f)?

File: content/git/concepts/commit/commit.md
Comment (Dusch4593) [inline content/git/concepts/commit/commit.md:43]: Awesome point! 👍

File: content/git/concepts/commit/commit.md
Comment (Dusch4593) [inline content/git/concepts/commit/commit.md:47]:
```suggestion
```shell
```

File: content/git/concepts/commit/commit.md
Comment (Dusch4593) [inline content/git/concepts/commit/commit.md:40]:
```suggestion
$ git commit
```

File: content/git/concepts/commit/commit.md
Comment (Dusch4593) [inline content/git/concepts/commit/commit.md:30]:
```suggestion
$ git add filename
```

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:23]:
```suggestion
```pseudo
```

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:29]:
```suggestion
In the example below, `git clone` is run to clone the repository for [Codecademy Docs](https://github.com/codecademy/docs):
```

File: content/git/concepts/init/init.md
Comment (Dusch4593) [inline content/git/concepts/init/init.md:25]:
```suggestion
### Step 1: Navigate to the Desired Directory
```

File: content/git/concepts/init/init.md
Comment (Dusch4593) [inline content/git/concepts/init/init.md:27]:
```suggestion
The `init` command initializes a Git repository in the current directory. Prior to using this command, users must take care to navigate to the appropriate directory &mdash; this will become the root directory of the repository.

The `cd` command can be used to navigate the filesystem from the command line:
```

If using an em dash, let's surround them with a single space.

File: content/git/concepts/init/init.md
Comment (Dusch4593) [inline content/git/concepts/init/init.md:30]:
```suggestion
$ cd go/to/desired/project/directory
```

File: content/git/concepts/init/init.md
Comment (Dusch4593) [inline content/git/concepts/init/init.md:38]:
```suggestion
$ git init
```

File: content/git/concepts/init/init.md
Comment (Dusch4593) [inline content/git/concepts/init/init.md:20]:
```suggestion
$ git init
```

File: content/git/concepts/init/init.md
Comment (Dusch4593) [reply content/git/concepts/init/init.md:25]: What do we think about replacing "appropriate directory" with "desired directory"?

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:24]:
```suggestion
git clone https://github.com/repository/URL.git
```

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:32]:
```suggestion
$ git clone https://github.com/Codecademy/docs.git
```

Purely a stylistic thing.

File: content/git/concepts/add/add.md
Comment (aedwardg) [reply content/git/concepts/add/add.md:57]: Yeah, I think that makes more sense here.

File: content/git/concepts/branch/branch.md
Comment (aedwardg) [reply content/git/concepts/branch/branch.md:46]: I don't know about having the `$` in front of the comments. Those commented lines are not commands that should be entered into the command line. If anything, maybe it is better that they get moved out of the code snippet altogether if each line has to be preceded by `$`

File: content/git/concepts/branch/branch.md
Comment (aedwardg) [reply content/git/concepts/branch/branch.md:53]: Sure, no problem.

Regarding `git switch`, that might be better added as a separate entry in a new PR, if at all. `git switch` and `git restore` are still considered experimental and I don't personally know anyone that uses them, but they have been around for a while now, so maybe it makes sense to add them.  If nothing else, it may help clear up some confusion about how to use `checkout`.

File: content/git/concepts/commit/commit.md
Comment (aedwardg) [reply content/git/concepts/commit/commit.md:30]: If we're changing this to filename and keeping the \`\`\`shell designation, should we add a file extension to this example?

File: content/git/concepts/init/init.md
Comment (aedwardg) [reply content/git/concepts/init/init.md:25]: Agree, "desired" is better.

File: content/git/concepts/init/init.md
Comment (aedwardg) [reply content/git/concepts/init/init.md:30]: Shouldn't this be a \`\`\`pseudo with no `$`?

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [reply content/git/concepts/branch/branch.md:53]: That sounds good! We should consider those in the near future but not for this PR. 😄

File: content/git/concepts/init/init.md
Comment (Dusch4593) [reply content/git/concepts/init/init.md:30]: I agree, yes, it should be. 😄

**Comment by Dusch4593** [comment]
Hey @aedwardg may I ask you to make this change to all 11 files: Revise each meta-`Description` property to be 150 characters or less. This is length is considered to make the SEO-friendly. 

As a guide, sometimes I like to use the first 1-2 sentences of the entry's opening paragraph but it's totally your call. 😄

**Comment by aedwardg** [comment]
@Dusch4593 Sure, no problem.  FYI though, Markdown Lint doesn't pass if I add those dollar signs on the commands with no output, so I'm going to leave them off for now.
image

**Comment by Dusch4593** [comment]
> @Dusch4593 Sure, no problem.  FYI though, Markdown Lint doesn't pass if I add those dollar signs on the commands with no output, so I'm going to leave them off for now.
> 
> image

Agreed

**Review by Dusch4593 — APPROVED** [review]
@aedwardg Sorry forgot to update this. Everything looks good to proceed to the next review! 👍 

cc: @yangc95 or @KTom101

File: content/git/concepts/clone/clone.md
Comment (aedwardg) [inline content/git/concepts/clone/clone.md:17]: @KTom101 Is there a reason you removed the semicolon and added backticks around the em dash here? Is that a specific syntax for Codecademy's markdown renderer? Because as far as I know that won't be rendered correctly with standard markdown.

**Review by KTom101 — COMMENTED** [review]
@aedwardg done with the second review.

File: content/git/concepts/add/add.md
Comment (yangc95) [reply content/git/concepts/add/add.md:52]: ⏫ +1 on this typo

**Review by yangc95 — COMMENTED** [review]
Hi @aedwardg, great to have you on Docs ☺️

Thanks for making edits to all these Git entries!

I thought I would help speed up the process for getting these entries up to standard, so I put in my review as well.

For future reference, it's ideal to keep changes to a smaller number of files for ease of reviewing. LMK if you want to discuss anything by tagging me and please tag me+@KTom101 once you're finished looking over our comments/suggestions!

Thanks a bunch 🍇

File: content/git/concepts/push/push.md
Comment (aedwardg) [reply content/git/concepts/push/push.md:41]: I agree this sentence isn't great. I'm going to update it in both files for this PR, but I also think in the future adding an entry for `git remote` and linking to it here would be better.

**Comment by aedwardg** [comment]
@yangc95 ♻️

**Review by yangc95 — APPROVED** [review]
I'm going ahead and approving this. @KTom101, are there any more improvements you have in mind? 🧠

File: content/git/concepts/branch/branch.md
Comment (yangc95) [reply content/git/concepts/branch/branch.md]: I see your point. Alright, then let's stick to the strict terminologies that Git's docs use 👍

File: content/git/concepts/branch/branch.md
Comment (Dusch4593) [inline content/git/concepts/branch/branch.md:54]: Could we also add a short bit about removing the remote branch equivalent (if one exists)?

```
git push <name-of-remote> --delete <name-branch>
```

Not sure if it should go here b/c it deals with branching or somewhere in `push.md` since this uses the `push` command.

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:21]:
```suggestion
The syntax for cloning in Git via HTTPS is this:
```

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:29]:
```suggestion
In the example below, `git clone` is used to clone the [Codecademy Docs](https://github.com/codecademy/docs) repository:
```

File: content/git/concepts/clone/clone.md
Comment (Dusch4593) [inline content/git/concepts/clone/clone.md:17]:
```suggestion
Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on a local machine. Here's a command that will interact with a remote branch &mdash; a branch on a server that can be accessed through the internet.
```

Would it be more accurate to say "repository" instead of "branch"? Or maybe "clone or branch" if `git clone` can clone from a specific branch?

File: content/git/concepts/pull/pull.md
Comment (Dusch4593) [inline content/git/concepts/pull/pull.md:21]:
```suggestion
The syntax for pulling changes in Git is this:
```

File: content/git/concepts/pull/pull.md
Comment (Dusch4593) [inline content/git/concepts/pull/pull.md:53]:
```suggestion
At this stage, pulling can accomplished with the names of a valid remote and branch:
```

File: content/git/concepts/push/push.md
Comment (Dusch4593) [inline content/git/concepts/push/push.md:50]:
```suggestion
At this stage, pushing can accomplished with the names of a valid remote and branch:
```

File: content/git/concepts/pull/pull.md
Comment (yangc95) [reply content/git/concepts/pull/pull.md:53]: Edits:
- can be accomplished*
- "the names of a valid remote and branch:" -> "a valid remote and branch name:"

File: content/git/concepts/push/push.md
Comment (yangc95) [reply content/git/concepts/push/push.md:50]: can be accomplished*

**Comment by KTom101** [comment]
On line 58 in the reset.md file there is an extra "the". It's not one of the lines that is a part of the pull request but it should be removed.

**Comment by yangc95** [comment]
Thanks @Dusch4593 and @KTom101 for your input! Let us know @aedwardg (when you get the chance) about the status of this entry. Hopefully after this next set of improvements, the PR will be ready to merge 🙏

**Comment by yangc95** [comment]
> ```
> git branch -m <old-branch-name> <new-branch-name>
> ```

@Dusch4593 were you thinking about adding that in the first block of the examples? Then in the second one, we'd change the branch name to the renamed one?

**Comment by Dusch4593** [comment]
> @Dusch4593 were you thinking about adding that in the first block of the examples? Then in the second one, we'd change the branch name to the renamed one?

@yangc95 Perhaps? I was thinking that we could add a brief section called "Renaming a Branch" in between the "Creating a Branch" and "Deleting a Branch" sections.

**Comment by yangc95** [comment]
> @yangc95 Perhaps? I was thinking that we could add a brief section called "Renaming a Branch" in between the "Creating a Branch" and "Deleting a Branch" sections.

Oh, that seems like a good spot as well 👀

**Comment by KTom101** [comment]
That’s a very good idea @Dusch4593

File: content/git/concepts/branch/branch.md
Comment (aedwardg) [reply content/git/concepts/branch/branch.md:54]: Holding off on this one for now. In the future, it might be worth adding, perhaps side by side with the way to do it on GitHub.

**Comment by aedwardg** [comment]
@yangc95 @Dusch4593 @KTom101 ♻️ 
Requested changes are done

**Comment by Dusch4593** [comment]
Merged! Thank you again, @aedwardg, for this solid contribution to our entire collection of Git entries. After the changes have settled in the live pages, if you profile is configured for credit, it will appear under the "Contributors" list. 😄

**Side-note:** Although I think this was mentioned earlier in this PR thread, I wanted to reiterate that it would be best to work with max of three entries per PR. This way, both contributors and maintainers get a moderate workload and the PR will generally take less time to review and merge.

**Pull request merged**
